packages/saber-theme-portfolio/src/layouts/post.vue

```diff
@@ -69,16 +74,16 @@ export default {
           content: `@${this.$themeConfig.twitter}`
         },
         {
-          name: 'og:title',
+          property: 'og:title',
           content: title
         },
         {
-          name: 'og:description',
+          property: 'og:description',
           content: description
         },
-        this.page.attributes.assets.cover && {
-          name: 'og:image',
-          content: this.page.attributes.assets.cover
+        {
+          property: 'og:image',
+          content: image
```

Comment:
Simply `this.page.attributes.assets.cover || this.$themeConfig.profilePicture`

Reply:
Let's add some docs in readme for `assets.cover` as well.

Reply:
Added